data/formats-data.js

```diff
@@ -3909,7 +3916,7 @@ exports.BattleFormatsData = {
 		tier: "OU",
 	},
 	glameow: {
-		randomBattleMoves: ["fakeout", "uturn", "suckerpunch", "hypnosis", "quickattack", "return", "foulplay"],
+		randomBattleMoves: ["fakeout", "uturn", "suckerpunch", "playrough", "quickattack", "return", "knockoff", "hiddenpowerfire"],
```

Comment:
keep hypnosis over hp fire

Reply:
sleeping steels is fine i guess...

Reply:
you're not doing much against steels with glameow regardless lol